apps/api/env/.env.mainnet

```
@@ -1,4 +1,4 @@
 REST_API_NODE_URL=https://consoleapi.akashnet.net
 RPC_NODE_ENDPOINT=https://consolerpc.akashnet.net
 REST_API_NODE_URL=https://consoleapi.akashnet.net
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The REST_API_NODE_URL key is duplicated

(DuplicatedKey)